### PR TITLE
Highlight days with events (new CalendarView class)

### PR DESCRIPTION
**Pull request by JadedCtrl**
Had a go at event-highlighting. Not sure if I've gotten everything right style-wise, but it does work.

### CODE REVIEW

File: src/OurCalendarView.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+
+void
+OurCalendarView::DrawDay (BView* owner, BRect frame, const char* text,
+	bool isSelected, bool isEnabled, bool focus, bool highlight)
+{
+	int drawnYear  = Date().Year();
+	int drawnMonth = Date().Month();
+	int drawnDay   = atoi(text);
```
Comment (humdingerb) [inline src/OurCalendarView.cpp:56]: Needs <stdlib.h> included.

File: src/OurCalendarView.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/*
```
Comment (humdingerb) [inline src/OurCalendarView.cpp:1]: Needs to be added to the Makefile. You may want to alpha-sort the source files, while you're at it.

File: src/OurCalendarView.cpp (hunk 1 above)
Comment (JadedCtrl) [reply src/OurCalendarView.cpp:57]: Heck, I forgot to git add the Makefile, thanks. Just did that and included stdlib in a new commit.

**Comment by waddlesplash** [comment]
Probably just "CalendarView" would be a better name for the subclass, IMO.

**Comment by JadedCtrl** [comment]
> Probably just "CalendarView" would be a better name for the subclass, IMO.

Honestly *true*, Our is hecking ugly.

File: src/CalendarView.h [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 Jadedctrl
+ * All rights reserved. Distributed under the terms of the MIT license.
+ */
+#ifndef OUR_CALENDAR_VIEW_H
+#define OUR_CALENDAR_VIEW_H
```
Comment (humdingerb) [inline src/CalendarView.h:6]: Last remnants or "Our". :)

Comment (JadedCtrl) [reply src/CalendarView.h]: Got it c:

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**